app/src/main/java/org/fossasia/openevent/app/contract/model/UtilModel.java

```diff
@@ -2,10 +2,20 @@
 
 public interface UtilModel {
```

Comment:
We need to mock interactions, there is no better way to do so than to create interface of it. And besides, it is always better to program in abstractions